### DISCUSSION TITLE
Unselect line from down

### DISCUSSION THREAD

**Discussion by Proful** [helix-editor/helix#5548] [Q&A]
x is for select line downward.

If I select 3 line & want to deselect last line how to do it?

**Comment by ghost** [comment]
* switch to visual mode `v`
* move up `k`
* sometimes you will need to press `x` again.

**Reply by Proful** [reply]
Thanks. it works, but not elegant.

**Reply by ghost** [reply]
Also you can remap key to this command:
``` toml
[keys.normal]
"A-q" = "extend_line_up"
```
And last, if you want to do this command but don't want to make remap, you can press `<Space>?` in editor, this will open command picker. Type any command you want to execute, e.g. `extend line`.

**Comment by 71GA** [comment]
@Proful The best way that I know of is to first press `v` to enter visual mode. At this point you can use arrow keys to move up/down, left/right and selection will precisely follow. Now press `x` to enter the "line mode" and now you can simply navigate entire line selection up/down with arrow keys.
